src/base/language_support.c

```diff
@@ -5830,8 +5839,6 @@ lang_init_coll_Utf8_tr_cs (LANG_COLLATION * lang_coll)
       lang_Next_alpha_char_TR_ti[i] = i + 1;
     }
 
-  assert (DIM (special_lower_cp) == DIM (special_upper_cp));
```

Comment:
why this assert is removed?

Reply:
It's a wrong copied code. It exists at the middle of the function. I copied the code wrongly.

Reply:
There is the same line at the release/10.2 branch. It seems not the wrong copied code.

Reply:
Aha, I made a mistake.